com.unity.cinemachine/Runtime/Core/CinemachineVirtualCameraBase.cs

```diff
@@ -815,7 +815,7 @@ static OnDomainReload()
             {
 #if UNITY_2023_1_OR_NEWER
                 var vcams = FindObjectsByType<CinemachineVirtualCameraBase>
-                    (FindObjectsInactive.Include, FindObjectsSortMode.None);
+                    (FindObjectsInactive.Include, FindObjectsSortMode.InstanceID);
```

Comment:
Sorry, I was wrong to say earlier that the order need to be determinate here.  It doesn't.  Better to use the faster unsorted version.